backend/data_export/tests/test_task.py

```diff
@@ -1,3 +1,4 @@
+from cmath import nan
 import os
```

Comment:
I think the `nan` is mistakenly imported and can be removed XD

Reply:
yeah it's accidentally imported XD I've removed it